src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -8623,9 +8657,9 @@ void Player_Action_80843954(Player* this, PlayState* play) {
     }
 }
 
-static struct_80832924 D_808545DC[] = {
-    { 0, 0x4014 },
-    { 0, -0x401E },
+static AnimSfxEntry D_808545DC[] = {
+    { 0, ANIMSFX_DATA(ANIMSFX_TYPE_8, 20) },
+    { 0, -ANIMSFX_DATA(ANIMSFX_TYPE_8, 30) },
```

Comment:
```suggestion
static AnimSfxEntry D_808545DC[] = {
    { NA_SE_NONE, ANIMSFX_DATA(ANIMSFX_TYPE_8, 20) },
    { NA_SE_NONE, -ANIMSFX_DATA(ANIMSFX_TYPE_8, 30) },
```
As you mention in `Player_ProcessAnimSfxList`, all animSfx in the range `ANIMSFX_TYPE_5` to `ANIMSFX_TYPE_9` don't use the `entry->sfxId` at all, they use preset sfxId's from the functions they call. Even so, it's still a sfxId entry, just unused, so I can see motivation for both using `NA_SE_NONE` or leaving it as 0. Thoughts?

Reply:
I think its better to convey that it represents nothing at all, and not "a sound effect that happens to play nothing". Its just filler for the field that isnt being used at all

Reply:
To rephrase my point, having `NA_SE_NONE` says to me that I can put an actual sound effect there and have it play something. When that is not the case. And really they could have put any value there, like -1 for example. But they happened to put 0